### ISSUE TITLE
struct size differs error

### ISSUE THREAD

**Issue by langalex** [gocardless/gocardless-pro-ruby#18]
scenario: I'm completing a redirect flow, calling:
```
response = gocardless_client.redirect_flows.complete(flow_id,
      params: {session_token: session_token})
mandate = response.links.mandate
```

when calling `links.mandate` I'm getting an `ArgumentError: struct size differs` error:
```
Stacktrace (most recent call first):

  gocardless_pro/resources/redirect_flow.rb:91:in `initialize'
    ).new(*@links.sort.map(&:last))
  gocardless_pro/resources/redirect_flow.rb:91:in `new'
    ).new(*@links.sort.map(&:last))
  gocardless_pro/resources/redirect_flow.rb:91:in `links'
    ).new(*@links.sort.map(&:last))
  app/services/gocardless_service.rb:72:in `complete_redirect_flow'
    attributes[:mandate] = response.links.mandate
```

**Comment by langalex** [comment]
never mind. fixed in 1.0.1

**Comment by greysteil** [comment]
Sorry you ran into this @langalex - it's permanently fixed now, so adding new links to the API won't break this or future client version.
